jsons/Policies.json

```diff
@@ -415,14 +415,14 @@
 			},
 			{
 				"name": "Entrepreneurship",
-				"uniques": ["Maintenance on roads & railroads reduced by [50]%","Provides 2 Trade Routes"],
+				"uniques": ["[-50]% Maintenance on roads & railroads","Provides 2 Trade Routes"],
 				"requires": ["Silk Road"],
 				"row": 2,
 				"column": 1
 			},
 			{
 				"name": "Mercantilism",
-				"uniques": ["+[25]% yield from every [Land Trade Route (Food)]","+[25]% yield from every [Land Trade Route (Production)]","+[25]% yield from every [Sea Trade Route (Food)]","+[25]% yield from every [Sea Trade Route (Production)]","[Gold] cost of purchasing items in cities [-20]%","[+2 Science] from every [Mint]",
+				"uniques": ["[+25]% yield from every [Land Trade Route (Food)]","[+25]% yield from every [Land Trade Route (Production)]","[+25]% yield from every [Sea Trade Route (Food)]","[+25]% yield from every [Sea Trade Route (Production)]","[-20]% Gold cost of purchasing items in cities ","[+2 Science] from every [Mint]",
```

Comment:
I dont believe the item discount unique should be changed, though I could be wrong

Reply:
At the very least there's a trailing space in the unique, invalidating it.

Reply:
https://github.com/yairm210/Unciv/blob/master/docs/uniques.md#stat-cost-of-purchasing-items-in-cities-amount

Reply:
resolved in https://github.com/ravignir/RekMOD/pull/43/commits/d98b1105a0e0d4f0661c3954f3b1811a2899ab7c